docs/quickstarts/packet.md

```diff
@@ -5,262 +5,307 @@
+Sample output:
 
-Your configurations are stored in /home/imran/lokoctl-assets/mycluster
+```
+NAME                            STATUS   ROLES    AGE   VERSION
+lokomotive-demo-controller-0      Ready    <none>   33m   v1.17.4
+lokomotive-demo-pool-1-worker-0   Ready    <none>   33m   v1.17.4
+lokomotive-demo-pool-1-worker-1   Ready    <none>   33m   v1.17.4
+```
 
-Now checking health and readiness of the cluster nodes ...
+Verify all pods are ready:
 
-Node                                          Ready    Reason          Message
+```console
+kubectl get pods -A
+```
 
-mycluster-controller-0                        True     KubeletReady    kubelet is posting ready status
-mycluster-controller-1                        True     KubeletReady    kubelet is posting ready status
-mycluster-controller-2                        True     KubeletReady    kubelet is posting ready status
-mycluster-worker-pool-1-worker-0              True     KubeletReady    kubelet is posting ready status
-mycluster-worker-pool-1-worker-1              True     KubeletReady    kubelet is posting ready status
-mycluster-worker-pool-1-worker-2              True     KubeletReady    kubelet is posting ready status
+Verify you can access httpbin:
 
-Success - cluster is healthy and nodes are ready!
+```console
+kubectl -n httpbin port-forward svc/httpbin 8080
+
+# In a new terminal.
+curl http://localhost:8080/get
 ```
 
-Use the generated `kubeconfig` file to access the Kubernetes cluster and list nodes.
+Sample output:
 
-```console
-export KUBECONFIG=./lokomotive-assets/cluster-assets/auth/kubeconfig
-kubectl get nodes
+```
+{
+  "args": {}, 
+  "headers": {
+    "Accept": "*/*", 
+    "Host": "localhost:8080", 
+    "User-Agent": "curl/7.70.0"
+  }, 
+  "origin": "127.0.0.1", 
+  "url": "http://localhost:8080/get"
+}
 ```
 
 ## Using the cluster
 
-At this point you have access to the Kubernetes cluster and can use it!
-If you don't have Kubernetes experience you can check out the [Kubernetes
-Basics official
-documentation](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/)
-to learn about its usage.
+At this point you should have access to a Lokomotive cluster and can use it to deploy applications.
+
+If you don't have any Kubernetes experience, you can check out the [Kubernetes
+Basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/) tutorial.
 
-**Note**: Lokomotive sets up a pretty restrictive Pod Security Policy that
-disallows running containers as root by default, check the [Pod Security Policy
-documentation](../concepts/securing-lokomotive-cluster.md#cluster-wide-pod-security-policy)
-for more details.
+>NOTE: Lokomotive uses a relatively restrictive Pod Security Policy by default. This policy
+>disallows running containers as root. Refer to the
+>[Pod Security Policy documentation](../concepts/securing-lokomotive-cluster.md#cluster-wide-pod-security-policy)
+>for more details.
```

Comment:
The tutorial linked uses service type load balancer, that won't work without metalLB+contour. I'd mention that in the note too

Reply:
Which tutorial? I don't understand your intention.
Anyway, we state in "next steps" that real world apps may need ingress + LB and link to a relevant guide.